### PR TITLE
Task/tests type conversions

### PR DESCRIPTION
**Pull request by jpizagno**
@morazow 
   Feel free to comment on anything, and we can change it.  
What are your thoughts ?

### CODE REVIEW

**Comment by morazow** [comment]
Hello @jpizagno,

Great! Thanks for working on this! Going to review.

File: src/it/scala/com/exasol/spark/TypesSuite.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        .load()
+      val schemaTest = df.schema
+      val fields = schemaTest.toList
+      fields.foreach(field => {
+        val dataType = field.dataType // access DataType, which should not be mapped
+      })
+    } catch {
+      case _: IllegalArgumentException =>
+    }
+  }
```
Comment (morazow) [inline src/it/scala/com/exasol/spark/TypesSuite.scala]: Could you please change this to `intercept`? You can check for an example [here](https://github.com/EXASOL/spark-exasol-connector/blob/master/src/it/scala/com/exasol/spark/LoadSuite.scala#L126-L128).

Something like:

```scala
val df = spark.read
        .format("com.exasol.spark")

...

intercept[ExpectedException] {
  df.schema.foreach(_.dataType)
}
```

File: src/main/scala/com/exasol/spark/util/Types.scala [hunk 2]
```diff
@@ -42,7 +44,7 @@ object Types extends LazyLogging {
 
   /**
    * Maps a JDBC type [[java.sql.Types$]] to a Spark SQL [[org.apache.spark.sql.types.DataType]]
-   *
+   *Image stream 'atmoz-sftp' was marked for deletion.
```
Comment (morazow) [inline src/main/scala/com/exasol/spark/util/Types.scala]: typo?

File: src/it/scala/com/exasol/spark/TypesSuite.scala (hunk 1 above)
Comment (jpizagno) [reply src/it/scala/com/exasol/spark/TypesSuite.scala]: @morazow 
    done.

File: src/main/scala/com/exasol/spark/util/Types.scala (hunk 2 above)
Comment (jpizagno) [reply src/main/scala/com/exasol/spark/util/Types.scala]: @morazow yes a type.  removed.

**Comment by 3cham** [comment]
@jpizagno @jamespiz don't forget to squash the commits & config your correct github username ;)

**Comment by jpizagno** [comment]
@morazow 
   I squashed the commits, and reset my username.  There is now a merge conflict, which I do not have write permissions to fix.  Can you resolve the conflicts?

File: src/main/scala/com/exasol/spark/util/Types.scala (hunk 2 above)
Comment (morazow) [reply src/main/scala/com/exasol/spark/util/Types.scala]: @jpizagno this string again somehow ended up here :)

Other than that all looks good!

**Comment by morazow** [comment]
@jpizagno apart from the typo string all looks good. Please let me know once it is removed; then we can merge the pr.

**Comment by jpizagno** [comment]
@morazow 
   I removed the typo. my apologies for letting that slip through ... again.   The unit-tests and Integration tests are green.  Let me know if you have more comments or suggestions?

**Comment by morazow** [comment]
@jpizagno great! lgtm!
I am going to merge the pr soon. Thanks for working on this!

**Comment by jpizagno** [comment]
@morazow 
    Sorry about this, but during the squash my work-git and home-git account names were both included. Now it looks like there are too many authors (jamespiz=wrong and jpizagno=correct).   I will not mix this up in the future.

You can fix this following this guide:
https://help.github.com/articles/changing-author-info/
OLD_EMAIL="james.pizagno@otto.de"
CORRECT_NAME="jpizagno"
CORRECT_EMAIL="jpizagno@gmail.com"

**Comment by morazow** [comment]
Good morning @jpizagno ,

No worries! I have updated the commits.

All the Best

**Pull request merged**